lib/liveValidation/liveValidator.ts

```diff
@@ -231,16 +231,21 @@ export class LiveValidator {
     );
 
     const allSpecs: SwaggerSpec[] = [];
+    let specMap: Map<string, SwaggerSpec> = new Map();
```

Comment:
This’ll always add a map for live validation scenario. Is it possible to add swagger path to swaggerspec type?